vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -55,21 +56,18 @@ testImagesPulled() {
     return
   fi
 
-  imagesNotPulled=()
+  imagesToBePulled=$(echo $imagesToBePulled | jq ".ContainerImages" | jq .[] --monochrome-output --compact-output)
```

Comment:
also wondering if we should keep the list of images as an arg instead of relying on the file